### ISSUE TITLE
Xcode 14.0 RC (14A309) Cannot find type 'sqlite3_snapshot' in scope

### ISSUE THREAD

**Issue by gabors** [groue/GRDB.swift#1276]
GRDB 5.26, integrating via SPM

Using just released Xcode Version 14.0 (14A309) 
 
Multiple build errors in WALSnapshot.swift

<img width="957" alt="image" src="https://user-images.githubusercontent.com/3587589/189039270-83b21e3e-2d45-441c-bfff-80fb82cf81e2.png">

**Comment by gabors** [comment]
@groue Upon closer look this is when building for Mac Catalyst on macOS Ventura 13.0 Beta (22A5331f)

Building for iOS 16 Sim works ok. But on Mac Catalyst I get the error as per above.

**Comment by groue** [comment]
Hello @gabors,

A fix will ship soon, in GRDB 5.26.1 and GRDB 6.0.0.

I'm responsible for this compiler error: I have introduced a dependency on a future macOS SDK much too early. The Xcode 14 betas did not reveal my mistake, but Xcode 14 RC does, and does it hard.

I'll ship the fix as fast as I can. If you want a quick workaround, fork and perform the following change in WALSnapshot.swift:

```diff
-#if GRDBCIPHER || (GRDBCUSTOMSQLITE && !SQLITE_ENABLE_SNAPSHOT) || compiler(<5.7)
+#if os(macOS) || targetEnvironment(macCatalyst) || GRDBCIPHER || (GRDBCUSTOMSQLITE && !SQLITE_ENABLE_SNAPSHOT) || compiler(<5.7)
```

**Comment by gabors** [comment]
Hi @groue Thanks so much for the quick reply and for offering to fix it soon. Much appreciated.
I'm not in a huge rush and will wait patiently on the fix in 5.26.1.

Seems like every year the Xcode RC behaves completely differently from Betas. :)

**Comment by groue** [comment]
Thank you @gabors. Without you, I wouldn't have noticed the issue with Mac Catalyst 👍

**Comment by groue** [comment]
[GRBD v5.26.1](https://github.com/groue/GRDB.swift/tree/v5.26.1) has just shipped with the fix (#1248 was reverted on macOS and Mac Catalyst).
